esm-samples/README.md

```diff
@@ -2,59 +2,16 @@
 
 The projects in this directory are samples that integrate the [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) build of the ArcGIS API for JavaScript with various frameworks, module bundlers and build tools. 
 
-## Get started
+For additional information, see the [Build with ES modules](https://developers.arcgis.com/javascript/latest/es-modules/) Guide topic in the SDK.
 
-Install the modules into your project:
+## Get Started
 
-```js
-npm install @arcgis/core
-```
-
-Then use `import` statements to load individual modules.
-
-```js
-import WebMap from '@arcgis/core/Map';
-import MapView from '@arcgis/core/views/MapView';
-
-const map = new Map({
-  basemap: "topo-vector"
-});
-
-const view = new MapView({
-  container: "viewDiv", 
-  map: map 
-});
-```
-
-## Copy assets and set assetsPath
-
-Copy the API’s assets, which includes styles, images, fonts, and localization files from the `@arcgis/core/assets` folder to your build directory. A simple way to accomplish this is to configure an NPM script that runs during your build process. For example, use npm to install `ncp` and configure a script in `package.json` to copy the assets folder. Here’s a React example:
-
-```js
-
-// package.json
-{
-    "script": {
-        "start": "npm run copy && react-scripts start",
-        "build": "npm run copy && react-scripts build",
-        "copy": "ncp ./node_modules/@arcgis/core/assets ./assets"
-    }
-}
-```
-
-Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) to insure the assets are correctly resolved, for example:
-
-```js
-import esriConfig from '@arcgis/core/config.js';
-
-esriConfig.assetsPath = './assets';
-```
+**Step 1** - Run `npm install`. 
 
-## Configure CSS
+**Step 2** Configure CSS. Choose a [theme](https://developers.arcgis.com/javascript/latest/guide/styling/index.html#themes) then apply it to your css. Here's a React example:
```

Comment:
- Should this be more similar to https://developers.arcgis.com/javascript/latest/get-started/ and just show loading `esri/css/main.css` since using a theme is optional?
- I'm not clear what "then apply it to your css" is referring to

Reply:
At some point in the past we started recommending using a theme. I agree it's cleaner to just say apply the un-themed option. I'll update this across all the READMEs. There will be some variation within a few samples. Some samples are using a dark theme because it looks better with the dark map.

Reply:
We were using the word "apply" to mean you had to change the CSS url to match the theme.

Reply:
As discussed, I tightened up the language, and updated all the samples READMEs to use `light` css themed URL for consistency.